centreon/tests/e2e/features/Cloud-notifications/01-notification-create/index.ts

```diff
@@ -0,0 +1,310 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+import {
+  enableNotificationFeature,
+  notificationSentCheck,
+  setBrokerNotificationsOutput,
+  waitUntilLogFileChange
+} from '../common';
+import { checkHostsAreMonitored, checkServicesAreMonitored } from 'e2e/commons';
+
+import data from '../../../fixtures/notifications/data-for-notification.json';
+
+var globalResourceType = '';
+var globalContactSettings = '';
+
+before(() => {
+  cy.startWebContainer({ useSlim: false });
+  enableNotificationFeature();
+  setBrokerNotificationsOutput({
+    name: 'central-cloud-notifications-output',
+    configName: 'central-broker-master'
+  });
+});
+
+after(() => {
+  cy.stopWebContainer();
+});
+
+beforeEach(() => {
+  cy.setUserTokenApiV1();
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.loginByTypeOfUser({ jsonName: 'admin' });
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/notifications?page=1&limit=10*'
+  }).as('getNotifications');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/users?page=1*'
+  }).as('getUsers');
+});
+
+Given('a user with access to the Notification Rules page', () => {
+  cy.navigateTo({
+    page: 'Notifications',
+    rootItemNumber: 3,
+    subMenu: 'Notifications'
+  });
+});
+
+Given('the user is on the Notification Rules page', () => {
+  cy.url().should('include', '/configuration/notifications');
+});
+
+Given(
+  'a {string} with hosts and {string}',
+  (resourceType: string, contactSettings: string) => {
+    globalResourceType = resourceType;
+    globalContactSettings = contactSettings;
+
+    switch (contactSettings) {
+      case 'a single contact':
+        cy.addContact({
+          name: data.contacts.contact1.name,
+          email: data.contacts.contact1.email,
+          password: data.contacts.contact1.password
+        });
+        break;
+      case 'two contacts':
```

Comment:
why do you add only contact2 in this case ?

Reply:
@kduret the container isn't restarted in this test, which means we already got contact_1

Reply:
Then, could you delete the contact at the end of the previous test, and create it again in this test
Because it makes the test less readable and add strong dependencies between tests